README.md

```diff
@@ -130,3 +130,48 @@ Overly complex structures and unnecessary code can be avoided by effective refac
 * Overly-nested and long functions
 * Long parameter lists
 
+## **Code Reviews**
+
+### Workflow
+
+Developers must use Git to collaborate on development projects. 
+
+To ensure we keep code processing short and make the most of our time both in development and as a team, it is important to follow submit structured work. Refer to the [git commands cheat sheet](https://gist.github.com/cferdinandi/ef665330286fd5d7127d) for a list of all commands that can be used for effective git-flow processes.
+
+Communication is key and team members are expected to prioritize each other's work to maintain a good workflow. 
+
+It is the responsibility of all team members to:
+
+* Allocate time for daily and weekly reviews
+* Communicate all feedback during reviews, this includes questions
+* Test before and during reviews to avoid bugs making it to production
+
+This example of [Crucible's team workflow](https://confluence.atlassian.com/crucible/defining-your-workflow-298977486.html) explains the various types of teams and code reviews. 
+
+
+![Crucible team workflow](https://confluence.atlassian.com/crucible/files/298977486/299958592/3/1346045693851/one-to-one-review.png)
+
+
+#### Plan the review
+
+* Identify the goal of the review - functionality, code quality, etc.
+* Focus on single functionality/ aspects of code to maintain focus
+
+Know what you are looking for in the code - what should be included/used and what should be avoided
+
+#### Review process
+
+The goal of a review is to
```

Comment:
Each time you lead with a point followed by bullet points you should add " : " to keep consistency, as above you have already done this.

For eg:
The goal of a review is to:

Reply:
done